src/NzbDrone.Core/Http/CloudFlare/CloudFlareCaptchaException.cs

```diff
@@ -0,0 +1,19 @@
+using NzbDrone.Common.Exceptions;
+using NzbDrone.Common.Http;
+
+namespace NzbDrone.Core.Http.CloudFlare
+{
+    public class CloudFlareCaptchaException : NzbDroneException
+    {
+        public HttpResponse Response { get; set; }
+
+        public CloudFlareCaptchaRequest CaptchaRequest { get; set; }
+
+        public CloudFlareCaptchaException(HttpResponse response, CloudFlareCaptchaRequest captchaRequest)
+            : base("Unable to access {0}, blocked by CloudFlare CAPTCHA. Likely due to shared-IP VPN.", response.Request.Url.Host)
```

Comment:
VPS mostly have their own static IP, so that's less of a problem.
We don't need to specify it explicitly, but the error is thrown and logged for all HTTP responses, not just Rarbg. So I thought it would be wise to point ppl to the likely suspect.